src/components/volunteer/VolunteerUser.tsx

```diff
@@ -28,6 +28,9 @@ export const VolunteerUser = ({
 }: { data: any } & StackScreenProps<any>) => {
   const [isMe, setIsMe] = useState<boolean>();
 
+  const { globalSettings } = useContext(SettingsContext);
+  const { navigation: navigationType } = globalSettings;
```

Comment:
please move these two lines to the top in this component as well. the existing use state line can follow as third line.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/7dd1b103cb0a5f045b70859f5356a5190615f456